docs/cudf/source/dask-xgb-10min.ipynb

```diff
@@ -174,56 +265,109 @@
+   "metadata": {},
+   "source": [
+    "#### How to run prediction via `xgb.dask.predict`\n",
     "\n",
     "1. `client`: the `dask.distributed.Client`\n",
-    "2. `bst`: the Booster produced by the XGBoost training session\n",
-    "3. `x_test`: an instance of `dask_cudf.DataFrame` containing the data to be inferenced (acquire predictions)\n",
-    "\n",
-    "`pred` will be an instance of `dask_cudf.Series`\n",
+    "2. `booster`: the Booster produced by the XGBoost training session\n",
+    "3. `X_test`: an instance of `dask_cudf.core.DataFrame` containing the data to be inferenced (acquire predictions)\n",
     "\n",
-    "We can use `dask.dataframe.multi.concat` to construct a `dask_cudf.DataFrame` by concatenating the list of `dask_cudf.Series` instances (`[pred]`)\n",
+    "`pred` and `true` are instances of `dask_cudf.core.Series` object. We align the index of both the Series by resetting it through `reset_index`. To compute the root mean squared error(RMSE) we first compute difference squared between the `pred` and `true` series through the operation `squared_error = ((pred - true)**2)`\n",
     "\n",
-    "`test` is a `dask_cudf.DataFrame` object with a single column named `0` (e.g.) `test[0]` returns `pred`. Additionally, the root-mean-squared-error (RMSE) can be computed by constructing a new column and assigning to it the value of the difference between predicted and labeled values squared. This is encoded in the assignment `test['squared_error'] = (test[0] - y_test['x'])**2`.\n",
-    "\n",
-    "Finally, the mean can be computed by using an aggregator from the `dask_cudf` API. The entire computation is initiated via `.compute()`. We take the square-root of the result, leaving us with `rmse = np.sqrt(test.squared_error.mean().compute())`. Note: `.squared_error` is an accessor for `test[squared_error]`... Like so:"
+    "Finally, the mean is computed by using an aggregator from the `dask_cudf` API. The entire computation is initiated via `.compute()`. We take the square-root of the result, leaving us with `rmse = np.sqrt(squared_error.mean().compute())`."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 11,
    "metadata": {},
    "outputs": [],
    "source": [
-    "rmse = np.sqrt(test.squared_error.mean().compute())\n",
-    "print('rmse value:', rmse)"
+    "pred = pred.reset_index(drop=True)\n",
+    "true = true.reset_index(drop=True)"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 12,
    "metadata": {},
    "outputs": [],
-   "source": []
+   "source": [
+    "squared_error = ((pred - true)**2)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "rmse value: 2.291584656127447\n"
+     ]
+    }
+   ],
+   "source": [
+    "rmse = np.sqrt(squared_error.mean().compute())\n",
```

Comment:
We could probably get rid of `numpy` usage altogether in this entire notebook if you use `cupy.sqrt` here

https://docs-cupy.chainer.org/en/stable/reference/generated/cupy.sqrt.html